src/main/kotlin/dsm/wemeet/global/mail/service/SendMailService.kt

```
@@ -0,0 +1,58 @@
+        val helper = MimeMessageHelper(message, true, "UTF-8")
+
+        helper.setTo(cleanEmail)
+        helper.setSubject("[wemeet] 이메일 인증번호 확인")
+        helper.setFrom(InternetAddress(mailProperties.username, "wemeet"))
+
+        val content = loadEmailTemplate().replace("\${authCode}", authCode)
+
+        helper.setText(content, true)
+        try {
+            mailSender.send(message)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)